### DISCUSSION TITLE
How to return timestampz as unix epoch using supabase sdk?

### DISCUSSION THREAD

**Discussion by techbabytalk** [supabase/supabase#28855] [Questions]
I want to return timestampz value to unix epoch for sorting purpose at client side. I have following code, which needs to be return dates in unix epoch format.

`const { data, error } = await supabase
  .from('countries')
  .select('updated_at')`

here updated_at is coming as string "2024-08-20T11:02:08.60973+00:00" and I want it to be like 1724478567.

**Comment by GaryAustin1** [comment]
I glanced and did not see any asking/answering of this before.
I can't think of anything to cast with.
If you can't convert in the client code with date functions then you could use a PostgREST computed field:  https://docs.postgrest.org/en/latest/references/api/computed_fields.html
